tests/phpunit/tests/query/commentFeed.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * @group query
+ */
+class Tests_Query_CommentFeed extends WP_UnitTestCase {
+	public static $post_type = 'post'; // Can be anything.
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		for ( $i = 0; $i < 10; $i ++ ) {
+			$post_id = $factory->post->create(
+				array(
+					'post_type'   => self::$post_type,
+					'post_status' => 'publish',
+				)
+			);
+			$factory->comment->create_post_comments( $post_id, 5 );
+		}
```

Comment:
```suggestion
		$post_ids = $factory->comment->create_many( 3 );
		foreach ( $post_ids as $post_id ) {
			$factory->comment->create_post_comments( $post_id, 5 );
		}
```

Perf: I reckon we can get away with 3 x 5 = 15 comments, maybe even less. The test case can be proven with less data. 
Opinionated: I reckon this is a little easier to read. 
Nit: No need to specify published posts, that's the default.

Reply:
Fixed